src/PharoThings-Devices-HD44780.package/PotLCD.class/instance/setPinsToDigitalOutput.st

```diff
@@ -1,5 +1,5 @@
 initialization
 setPinsToDigitalOutput
 	| gpioArray |
-	gpioArray := (gpioDic values collect: #asGlamorousArray ) flattened.
+	gpioArray := ({ clockPin . modePin . dataPins . backlightPin } collect: #asGlamorousArray ) flattened.
```

Comment:
Forgot to ask last time why asGlamorousArray is needed here? I think #flattened will work just for {} collection.
It is better to avoid Glamorous dependency. Otherwise minimal Pharo server will require all that code